install.sh

```
@@ -64,4 +68,20 @@ add_to_shell_config "$HOME/.zshrc"
+export PATH="$CURRENT_DIR:$PATH"
+
+# Run update (plugin setup, changelog, etc.)
+echo "🔄 Running tools update..."
+tools update
+
+echo ""
+echo "🎉 Setup complete."
+
+if [ "$SHELL_CONFIG_CHANGED" = true ]; then
+    case "$SHELL" in
+        */zsh)  echo "   Sourcing ~/.zshrc..."; source "$HOME/.zshrc" ;;
```

Shellcheck (0.11.0)

[info] Not following: ./.zshrc was not specified as input (see shellcheck -x).

(SC1091)